tests/test_models_circuit.py

```diff
@@ -42,6 +43,28 @@ def test_circuit_init():
     assert c.nqubits == 2
 
 
+def test_eigenstate():
+    nqubits = 3
+    c = Circuit(nqubits)
+    c.add(gates.M(*list(range(nqubits))))
+    c2 = initialize(nqubits, eigenstate="-")
+    assert c(nshots=100, initial_state=c2).frequencies() == {"111": 100}
+    c2 = initialize(nqubits, eigenstate="+")
+    assert c(nshots=100, initial_state=c2).frequencies() == {"000": 100}
```

Comment:
Out of personal curiosity: where is this mapping between `+/-` and `0/1` defined?

Reply:
In case it is supposed to be defined by the `initialize` function, please document in the related docstring.

If it is a more general Qibo convention, consider referencing in the said docstring the place where this convention is declared.

Otherwise, there is also the option of dropping `+/-` in favor of explicit `0/1`, and drop one translation layer.

Reply:
I agree that `+` and `-` are conventional, but `0` and `1` are not, so the mapping is still arbitrary.

Reply:
Indeed, the mapping is not conventional (at least I don't find it easily online), it was my bias due to the QM lectures I followed as a student. I will update the docs.